server/server/settings.py

```diff
@@ -162,3 +163,10 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = '/static/'
+
+
+# Cors Settings
+CORS_ALLOW_CREDENTIALS = True
+
+# Below needs to be changed for production
+CORS_ORIGIN_WHITELIST = ['http://localhost:3000']
```

Comment:
I'll add an example, that's a great suggestion. 

Basically this would be wherever your deploy your react application, I'll definitely add a sample production url (which I think is what you're looking for on this). I might be misunderstanding what you're asking for here, so please let me know if that's the case.

Reply:
I believe that's exactly right, just adding a production level domain. Since there's no separation between settings files for different environments, I guess it's fine to leave a comment rather than adding it to the list.